llama_stack/logcat.py

```diff
@@ -109,10 +109,16 @@ def format(self, record):
 def init(default_level: int = logging.INFO) -> None:
     global _default_level, _category_levels, _logger
 
+    log_file = os.environ.get("LLAMA_STACK_LOG_FILE")
```

Comment:
add quick doc on how `LLAMA_STACK_LOG_FILE` will be used?

Reply:
will do!